### PR TITLE
Move Book's Admin functions to Admin::BooksController

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
https://github.com/smashingboxes/apprenticeship/tree/master/Curriculum/Backend/bookstore
Because only admins should be changing what books are listed on the site

### What Changed?
All BookController actions except index and show were moved to Admin namespace
Tests were updated for login functionality

### CODE REVIEW

File: app/views/admin/books/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<%= render 'filters' %>
```
Comment (dacur) [inline app/views/admin/books/index.html.erb]: partials ftw :+1:

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>The Ugly Bookstore</title>
+    <title>The Beautiful Rails Bookstore</title>
```
Comment (dacur) [inline app/views/layouts/application.html.erb]: :100:

**Comment by dacur** [comment]
looks good to me, i'm assuming all of your tests are passing

:shipit:

**Comment by TylerRockwell** [comment]
They are. Thanks for the cr, @dacur !

**Pull request merged**